pkg/http/github.go

```diff
@@ -40,11 +40,11 @@ func getLatestVersion() (string, error) {
 		return "", err
 	}
 
-	if version, exists := body["tag_name"]; exists {
+	if version, exists := body["version"]; exists {
 		return version.(string), nil
 	}
 
-	return "", errors.New("unable to retrieve tag_name of latest release")
+	return "", errors.New("unable to retrieve 'version' of latest release")
```

Comment:
This is probably a good opportunity to make this message a little more user friendly

```suggestion
	return "", errors.New("unable to retrieve latest CLI version")
```

Reply:
I'm game!